Cargo.lock

```diff
@@ -671,205 +342,178 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "proc-macro2",
+ "rand_core 0.4.2",
 ]
 
 [[package]]
-name = "radium"
-version = "0.7.0"
+name = "rand_core"
+version = "0.4.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "dc33ff2d4973d518d823d61aa239014831e521c75da58e3df4840d3f47749d09"
+checksum = "9c33a3c44ca05fa6f1807d8e6743f3824e8509beca625669633be0acbdf509dc"
 
 [[package]]
-name = "rand"
-version = "0.8.5"
+name = "rand_hc"
+version = "0.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "34af8d1a0e25924bc5b7c43c079c942339d8f0a8b57c39049bef581b46327404"
+checksum = "7b40677c7be09ae76218dc623efbf7b18e34bced3f38883af07bb75630a21bc4"
 dependencies = [
- "libc",
- "rand_chacha",
- "rand_core",
+ "rand_core 0.3.1",
 ]
 
 [[package]]
-name = "rand_chacha"
-version = "0.3.1"
+name = "rand_isaac"
+version = "0.1.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "e6c10a63a0fa32252be49d21e7709d4d4baf8d231c2dbce1eaa8141b9b127d88"
+checksum = "ded997c9d5f13925be2a6fd7e66bf1872597f759fd9dd93513dd7e92e5a5ee08"
 dependencies = [
- "ppv-lite86",
- "rand_core",
+ "rand_core 0.3.1",
 ]
 
 [[package]]
-name = "rand_core"
-version = "0.6.4"
+name = "rand_jitter"
+version = "0.1.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ec0be4795e2f6a28069bec0b5ff3e2ac9bafc99e6a9a7dc3547996c5c816922c"
+checksum = "1166d5c91dc97b88d1decc3285bb0a99ed84b05cfd0bc2341bdf2d43fc41e39b"
 dependencies = [
- "getrandom",
+ "libc",
+ "rand_core 0.4.2",
+ "winapi",
 ]
 
 [[package]]
-name = "rayon"
-version = "1.10.0"
+name = "rand_os"
+version = "0.1.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "b418a60154510ca1a002a752ca9714984e21e4241e804d32555251faf8b78ffa"
+checksum = "7b75f676a1e053fc562eafbb47838d67c84801e38fc1ba459e8f180deabd5071"
 dependencies = [
- "either",
- "rayon-core",
+ "cloudabi",
+ "fuchsia-cprng",
+ "libc",
+ "rand_core 0.4.2",
+ "rdrand",
+ "winapi",
 ]
 
 [[package]]
-name = "rayon-core"
-version = "1.12.1"
+name = "rand_pcg"
+version = "0.1.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "1465873a3dfdaa8ae7cb14b4383657caab0b3e8a0aa9ae8e04b044854c8dfce2"
+checksum = "abf9b09b01790cfe0364f52bf32995ea3c39f4d2dd011eac241d2914146d0b44"
 dependencies = [
- "crossbeam-deque",
- "crossbeam-utils",
+ "autocfg 0.1.8",
+ "rand_core 0.4.2",
 ]
 
 [[package]]
-name = "reddsa"
-version = "0.5.1"
+name = "rand_xorshift"
+version = "0.1.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "78a5191930e84973293aa5f532b513404460cd2216c1cfb76d08748c15b40b02"
+checksum = "cbf7e9e623549b0e21f6e97cf8ecf247c1a8fd2e8a992ae265314300b2455d5c"
 dependencies = [
- "blake2b_simd",
- "byteorder",
- "group",
- "hex",
- "jubjub",
- "pasta_curves",
- "rand_core",
- "serde",
- "thiserror",
- "zeroize",
+ "rand_core 0.3.1",
 ]
 
 [[package]]
-name = "redjubjub"
-version = "0.7.0"
+name = "rdrand"
```

Comment:
This is apparently only used on SGX, which is good by me because I don't trust it (it's an obvious backdoor target, even when it is working correctly; see https://en.wikipedia.org/wiki/RDRAND#Reception and https://arstechnica.com/gadgets/2019/10/how-a-months-old-amd-microcode-bug-destroyed-my-weekend/).
```
$ for target in $(rustc --print target-list); do echo ${target} && cargo tree --target ${target} -i rdrand 2>nul; done
[...]
x86_64-fortanix-unknown-sgx
rdrand v0.4.0
└── rand_os v0.1.3
    └── rand v0.6.5
        └── proptest v0.9.6
            [dev-dependencies]
            └── zcash_script v0.2.0 ([...]/zcash_script)
```